ai_diffusion/settings.py

```diff
@@ -75,7 +76,7 @@ class Settings(QObject):
     )
 
     server_backend: ServerBackend
-    _server_backend = Setting("Server Backend", ServerBackend.cuda)
+    _server_backend = Setting("Server Backend", ServerBackend.cpu)
```

Comment:
ServerBackend.default() maybe a good idea if you prefer it, though I think it is not too hard to ask people to switch to GPU when they have it rather than assuming there is always cuda on windows/linux.

Use cuda as default breaks when cuda is filtered out on mac.

Reply:
I think majority will run cuda (just from gpu distribution and how unpractical cpu mode is), so it should be default for win/linux. It's also annoying if you only install CPU and then want to switch (it won't work), but the reverse is not a problem.
For Mac I don't know what happens if you select MPS without M1/M2 hardware, but it's not as critical since the install is the same for CPU anyway.

Reply:
Added ServerBackend.default(), returning ServerBackend.mps or ServerBackend.cuda based on util.is_macos.
Please review.